supervision/detection/tools/inference_slicer.py

```diff
@@ -1,6 +1,6 @@
 import warnings
 from concurrent.futures import ThreadPoolExecutor, as_completed
-from typing import Callable, Optional, Tuple, Union
+from typing import Callable, Optional, Union
```

Comment:
lets replace `Union` with `|` and `Optional` with `| None` to keep things consistent